bottlecap/src/logs/datadog.rs

```diff
@@ -0,0 +1,48 @@
+use std::error::Error;
+
+use tracing::{debug, error};
+
+use crate::logs::processor::IntakeLog;
+
+pub struct DdApi {
```

Comment:
Maybe give this a more descriptive name than DdApi. Or just call it Api, and use the fact that it's namespaced under logs.